_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -15,6 +15,14 @@
 // Load WP_Error for error messages.
 require_once ABSPATH . '/wp-includes/class-wp-error.php';
 
+// Load API endpoint base classes
+require_once dirname( __FILE__ ) . '/core-api/class.jetpack-core-api-endpoint.php';
+require_once dirname( __FILE__ ) . '/core-api/class.jetpack-core-api-xmlrpc-consumer-endpoint.php';
+require_once dirname( __FILE__ ) . '/core-api/class.jetpack-core-api-writable.php';
+
+// Load API endpoints
+require_once dirname( __FILE__ ) . '/core-api/class.jetpack-core-api-module-endpoints.php';
```

Comment:
We could use the `JETPACK__PLUGIN_DIR` constant for all these like it's done in other places of the file https://github.com/Automattic/jetpack/blob/master/_inc/lib/class.core-rest-api-endpoints.php#L503